CommonTools/RecoAlgos/interface/FQueue.h

```diff
@@ -0,0 +1,151 @@
+#ifndef COMMONTOOLS_RECOALGOS_FQUEUE_H
+#define COMMONTOOLS_RECOALGOS_FQUEUE_H
+
+#include <vector>
+
+
+template<class T>
+
+class FQueue
+{
+public:
+	FQueue()
+	{
+		theSize = 0;
+		theFront = 0;
+		theTail = 0;
+		theCapacity = 0;
+
```

Comment:
The same comments about empty lines and indentations also hold for this file (not marked all them)